nefertari/__init__.py

```diff
@@ -49,6 +49,11 @@ def includeme(config):
     config.add_request_method(get_resource_map, 'resource_map', reify=True)
 
     config.add_tween('nefertari.tweens.cache_control')
+    try:
+        import pyramid_tm
+        config.add_tween('nefertari.tweens.es_tween', over='pyramid_tm.tm_tween_factory')
+    except ImportError:
+        pass
```

Comment:
This was added for tests because `pyramid_tm` wasn't added as dependency to nefertari. This is hardcoded solution and I want to discuss with you better way to solve it.